src/Debug.js

```diff
@@ -34,3 +34,29 @@ exports._debugger = function (f) {
   debugger;
   return f();
 };
+
+var now = (function () {
+  var perf;
+  if (typeof performance !== "undefined") {
+    // In browsers, `performance` is available in the global context
+    perf = performance;
+  } else if (req) {
+    // In Node, `performance` is an export of `perf_hooks`
+    try { perf = req("perf_hooks").performance; }
+    catch(e) { }
+  }
+
+  if (perf) {
+    return (function() { return perf.now(); });
+  } else {
+    return (function() { return new Date(); });
```

Comment:
I guess this works just now because `valueOf` for `Date` turns it into milliseconds? Maybe we should use `Date.now()` instead?